src/MeetingsSDKAdapter.js

```diff
@@ -166,11 +207,12 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    */
   createMeeting(target) {
     return from(this.datasource.meetings.create(target)).pipe(
+      flatMap(({id, sipUri}) => from(this.fetchMeetingTitle(sipUri)).pipe(map((title) => ({ID: id, title})))),
```

Comment:
Would it be clearer to use `destination`? or does `sipUri` also takes Hydra IDs, even though they're not technically SIP URIs?

Reply:
yes